go/pvl/regs.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
```

Comment:
Any reason not to just call it "registers" so that people don't mistake it for "regulations" or "regulars"?